auth/index.html

```diff
@@ -1077,6 +1090,7 @@ <h2>Message log</h2>
           },
           false
         );
+        // also, reset isn't a completely accurate name for this
```

Comment:
I understand this comment but others may not? You can complete the thought by saying "because this only clears the key, without regenerating a fresh one"

Reply:
Good call!